package.json

```diff
@@ -10,7 +10,8 @@
     "test:ui": "vitest --ui",
     "test:coverage": "vitest run --coverage",
     "lint": "eslint --ignore-path .gitignore --fix src",
-    "prettier": "prettier src --write"
+    "prettier": "prettier src --write",
+    "deploy": "npm run build && cd cdk-app && npm run cdk deploy"
```

Comment:
A bit wrong script) It's linked to the `cdk deploy` npm-script in the `package.json` in the `cdk-app` folder. But this script doesn't exist.
I think this works just because the names of npm-script and cdk-deploy command are the same - `cdk`. But this is absolutely minor issue.

Reply:
I am not sure If I understoot what you mean under *linked to the cdk deploy npm-script* but I changed the script and removed duplicated `npm run` as it was not necessary:
```Json    
"deploy": "npm run build && cd cdk-app && cdk deploy"
```